backend/src/modules/doubles-tournament/core/application/TablesQueueRepository.ts

```diff
@@ -0,0 +1,10 @@
+import { TablesQueue } from '../domain/TablesQueue';
+import { QueuedTable } from '../domain/QueuedTable';
+
+export interface TablesQueueRepository {
+  save(tablesQueue: TablesQueue): Promise<void>;
+
+  findByTournamentId(tournamentId: string): Promise<TablesQueue | undefined>;
+
+  findFreeTablesByTournamentId(tournamentId: string): Promise<QueuedTable[] | undefined>;
```

Comment:
Takie metody jak te powinny być poza repozytorium. W repozytorium zapisujemy/wczytujemy jeden typ - w tym przypadku TablesQueue. Bo to jest nasza jednostka transakcji - na tym wykonujemy operację. Czyli albo wczytujemy i zapisujemy to w całości, albo w ogóle. 

Czy to coś Ci mówi / jest zrozumiałe?